readme.md

```diff
@@ -118,6 +118,7 @@ const limiter = rateLimit({
 	max: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes)
 	standardHeaders: true, // Return rate limit info in the `RateLimit-*` headers
 	legacyHeaders: false, // Disable the `X-RateLimit-*` headers
+	store: new ExternalStore(), // Use an external store for more precise rate limiting
```

Comment:
Hum, I'm a little iffy about this. I feel like `ExternalStore` is going to confuse some folks. Maybe the redis store?

Reply:
what about 
```suggestion
	// store: ... // Use an external store for more precise rate limiting
```